src/components/CommentButton/index.tsx

```diff
@@ -231,6 +231,9 @@ export const CommentButton: React.FC<Props> = ({ characterId, noteId, iconSize =
                       }}
                       onComment={comments.refetch}
                       onEdit={comments.refetch}
+                      closeBottomSheet={() => {
```

Comment:
Maybe the prop name onNavigateToUserProfile is clearer.

Reply:
It has been modified.